conf/machine/imx8ulp-9x9-lpddr4-evk.conf

```diff
@@ -0,0 +1,23 @@
+#@TYPE: Machine
+#@NAME: i.MX 8ULP 9x9 EVK
+#@SOC: i.MX8ULP
+#@DESCRIPTION: Machine configuration for NXP i.MX 8ULP 9x9 Evaluation Kit with LPDDR4
```

Comment:
These boards with `9x9` in the name seem to be only for internal use, so not upstreamed.